modules/nw-sriov-topology-manager.adoc

```diff
@@ -9,8 +9,14 @@ You can create a NUMA aligned SR-IOV pod by restricting SR-IOV and the CPU resou
 
 .Prerequisites
 
-* Install the OpenShift CLI (`oc`).
-* Enable a LatencySensitive profile and configure the CPU Manager policy to `static`.
+* You have installed the OpenShift CLI (`oc`).
+* You have configured the CPU Manager policy to `static`. For more information on CPU Manager, see xref:../../scalability_and_performance/using-cpu-manager.html#using-cpu-manager[Using CPU Manager].
```

Comment:
s/html/adoc/

Reply:
I have moved it to the existing Additional resources section and mentioned that the user should refer to that section for more information as SME was particular about giving a reference while listing the prerequisite